apps/skilavottord/web/screens/RecyclingCompanies/RecyclingCompanies.tsx

```diff
@@ -62,42 +63,52 @@ const RecyclingCompanies: FC = () => {
               title: `${sidenavText.companies}`,
               link: `${routes.recyclingCompanies.baseRoute}`,
             },
+            {
```

Comment:
Further on, this should only be available to recycling fund. So can probably remove.